### DISCUSSION TITLE
get user quota information to pass on my bash script

### DISCUSSION THREAD

**Discussion by niele84-dot** [drakkan/sftpgo#1446] [Q&A] [answered]
Hi,

I have my instance of SFTPGO set up to record video and images from my set of webcams but the disk space gets full very quickly and I want to remove older files to free up some space, up to a certain threshold.e.g. remove files to free up to 5% of space.

I can write a script myself in bash to remove the older files, but I did not find a way to retrieve the quota information for a user, in a way that can be passed on to the bash script.

Does anyone know how to do it?

**Comment by drakkan** [comment, marked as answer]
Hello, you can retrieve quota information using the SFTPGo REST API, take a look [here](https://sftpgo.stoplight.io/docs/sftpgo/774771a1bd72c-get-users)
